include/cling/Interpreter/Interpreter.h

```diff
@@ -712,6 +713,24 @@ namespace cling {
     ///
     void* getAddressOfGlobal(llvm::StringRef SymName, bool* fromJIT = 0) const;
 
+    ///\brief Get a given macro definition by name.
+    ///
+    ///\param[in]  Name - the name of the macro to look for
+    ///
+    ///\returns the MacroInfo if the macro was defined, otherwise null
+    ///
+    const clang::MacroInfo* getMacro(llvm::StringRef Name) const;
```

Comment:
Would it make sense to rename this getCppMacro ? (In some context macro is also sometimes synonym with script)